examples/appdir/tailwind.config.js

```diff
@@ -1,5 +1,7 @@
+const { withUt } = require("./node_modules/@uploadthing/react/src/tw");
```

Comment:
this should probably be a proper export

Reply:
I guess you meant import

And I tried but it looks like TW config does not support conventional imports with absolute paths. Only require with relative import works for me

Reply:
Well even just `require("uploadthing/tw")`. It should be supported since you can require plugins that way?

Reply:
When trying `require("@uploadthing/react/tw")` getting following error:
![image](https://github.com/pingdotgg/uploadthing/assets/55754540/fd2893a5-a169-45b2-a921-d9d5f28957cb)
Even though package.json has this subpath defined
<img width="426" alt="image" src="https://github.com/pingdotgg/uploadthing/assets/55754540/53ca369d-5a7f-4238-adb9-ffa570d3570d">
I checked Tailwind's [package.json](https://github.com/tailwindlabs/tailwindcss/blob/master/package.json) and as far as I can see they do not specify `exports` property at all.

 I will try to move Tailwind utility to separate package `@uploadthing/tw` (that's a good thing anyway) and play around with build configuration to make it work properly

And what do you mean by

> And `import` for ts configs would be great

Didn't get what you want me to do

Reply:
I was able to achieve proper import experience in tailwind.config
<img width="819" alt="image" src="https://github.com/pingdotgg/uploadthing/assets/55754540/a2134fcd-4fda-47ef-a7db-42bb0827ae66">

Reply:
I fixed the bundling issues in https://github.com/pingdotgg/uploadthing/pull/217/commits/ead7636a7d381cb941dbc784821e200dae1e9cfe

Reply:
Thanks for that